docs/download_files.md

```diff
@@ -105,6 +105,13 @@ The `single_cell_metadata.tsv` file is a tab-separated table with one row per li
 | `subdiagnosis`    | Subcategory of diagnosis or mutation status (if applicable)    |
 | `tissue_location` | Where in the body the tumor sample was located                 |
 | `disease_timing`  | At what stage of disease the sample was obtained, either diagnosis or recurrence |
+| `organism`         | The organism the sample was obtained from (e.g., `Homo_sapiens`) |
+| `development_stage_ontology_term_id` | `HsapDv` ontology term indicating developmental stage. `unknown` indicates age is unavailable. |
```

Comment:
These are all going to correspond to terms that are specific to the age value, right? Might as well be more specific.

Reply:
Link to ontology? http://obofoundry.org/ontology/hsapdv.html